### PR TITLE
[ADD] Added migration to recalculated warehouse on Inventory Adjustme…

### PR DESCRIPTION
**Pull request by KirillVorobey**
…nts and dependence of selected locations on warehouse

### CODE REVIEW

File: ventor_base/__init__.py [hunk 1]
```diff
@@ -11,6 +11,14 @@ def _post_init_hook(cr, registry):
     """
     env = api.Environment(cr, SUPERUSER_ID, {})
 
+    all_stock_inventory = env['stock.inventory'].search([
+            ('state', '=', 'draft'),
+            ])
```
Comment (ventor-dev) [inline ventor_base/__init__.py]: we need to also update non-draft inventories to hide them

File: ventor_base/migrations/14.0.1.3.6/end-set_warehouses_on_inventory.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import SUPERUSER_ID, _, api
+
+
+def migrate(cr, version):
+    """Update warehouse_id for all stock inventory"""
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    all_stock_inventory = env['stock.inventory'].search([
+            ('state', '=', 'draft'),
+            ])
```
Comment (ventor-dev) [inline ventor_base/migrations/14.0.1.3.6/end-set_warehouses_on_inventory.py]: we need to also update non-draft inventories to hide them

File: ventor_base/__init__.py (hunk 1 above)
Comment (KirillVorobey) [reply ventor_base/__init__.py]: Done

File: ventor_base/migrations/14.0.1.3.6/end-set_warehouses_on_inventory.py (hunk 2 above)
Comment (KirillVorobey) [reply ventor_base/migrations/14.0.1.3.6/end-set_warehouses_on_inventory.py]: Done

**Pull request merged**